tests/Feature/Service/ServiceWorkersPageTest.php

```diff
@@ -5,29 +5,73 @@
 use Tests\StoreTestCase;
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 use App\AdminUser;
-
+use App\Centre;
+use App\CentreUser;
+use Illuminate\Support\Collection;
 
 class ServiceWorkersPageTest extends StoreTestCase
 {
     use DatabaseMigrations;
 
     /** @var AdminUser $adminUser */
     private $adminUser;
+
+    /** @var CentreUser $userHomeCentreNoAlt */
+    private $userHomeCentreNoAlt;
+
+    /** @var CentreUser $userHomeCentreTwoAlt */
+    private $userHomeCentreTwoAlt;
+
+    /** @var CentreUser $downlaoderUser */
+    private $downloaderUser;
+
+    /** @var Centre $centre */
+    private $centre;
+
+    /** @var Collection $altCentres */
+    private $altCentres;
+
     private $workersRoute;
 
     public function setUp()
     {
         parent::setUp();
+
         $this->adminUser = factory(AdminUser::class)->create();
+        // Create 3 centres in an area
+        $this->centre = factory(Centre::class, 3)->create();
+        // Create 3 alt centres
+        $this->altCentres = factory(Centre::class, 2)->create([]);
         $this->workersRoute = route('admin.centreusers.index');
+
+        // Create 1 user with a home centre and no alternatives
+        $this->userHomeCentreNoAlt = factory(CentreUser::class)->create([
+            "name"  => "test user",
+            "email" => "testuser@example.com",
+        ]);
+        $this->userHomeCentreNoAlt->centres()->attach(1, ['homeCentre' => true]);
+
+        // Create 1 user with with a homeCentre and 2 alternatives
+        $this->userHomeCentreTwoAlt = factory(CentreUser::class)->create([
+            "name"  => "test user with alternatives",
+            "email" => "testuseralternatives@example.com",
+        ]);
+        $this->userHomeCentreTwoAlt->centres()->attach($this->altCentres->pluck('id')->all(), ['homeCentre' => true]);
```

Comment:
you've set homeCentre True on all centres. technically possible but dangerous, a user should only have one homeCentre.

1. create the user
2. add a single centre, with homeCentre=true
3. add a couple of atlernatives, without homeCentre